src/forms/TimeInput.jsx

```diff
@@ -56,24 +98,85 @@ class TimeInput extends React.Component {
 		return (
 			<span>
 				{ label && <label htmlFor={id} className={labelClassNames}>{label}</label> }
-				<input
-					id={id}
-					type='time'
-					name={name}
-					value={value}
-					className={classNames}
-					required={required}
-					onChange={this.onChange}
-					ref={ input => this.inputEl = input }
-					{...other}
-				/>
+				{
+					supportsTime
+					?
+						<input
+							id={id}
+							type='time'
+							name={name}
+							value={value}
+							className={classNames}
+							required={required}
+							onChange={this.onChange}
+							ref={ input => this.inputEl = input }
+							{...other}
+						/>
+					:
+						<div>
+							<Flex>
+								<FlexItem>
+									<div className="fauxInput">
+										<Flex>
+											<FlexItem shrink>
+												<input type="number"
+													id="hours"
+													name="hours"
+													min={this.state.meridian ? 1 : 0} // return number from merdian true/false?
+													max={getMaxHours(this.state.meridian)}
+													className="field--reset"
+													onChange={this.onHoursChange}
+													value={this.state.meridian ? this.state.hours : this.state.hours % 24} /> { /* this.state.hours % 24 */ }
+											</FlexItem>
+											<FlexItem shrink>
+												{':'}
+											</FlexItem>
+											<FlexItem shrink>
+												<input type="number"
+													id="minutes"
+													name="minutes"
+													min={0}
+													max={59}
+													className="field--reset"
+													onChange={this.onMinutesChange}
+													value={this.state.minutes} />
+											</FlexItem>
+										</Flex>
+									</div>
+								</FlexItem>
+								{ this.state.meridian &&
+									<FlexItem shrink>
+										<SelectInput
+											id='meridian'
+											name='meridian'
+											onChange={this.onMeridianChange}
+											options={[
+												{ label: 'PM', value: 'PM' },
+												{ label: 'AM', value: 'AM' }
+											]}
+										/>
+									</FlexItem>
+								}
+							</Flex>
+							<input
```

Comment:
also I think this input would have the actual `name` of the time input - since we would want to get the value here.